<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="funcTable"/>
     </item>
     <item row="0" column="2">
      <widget class="QTableWidget" name="musicTable"/>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>364</y>
          <width>46</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>41</width>
          <height>341</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>850</y>
      <width>741</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>正在播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="musicBrowser">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ImgDisp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
